src/app/home-page/home-logged-in/home-logged-in.component.html

```diff
@@ -13,10 +13,148 @@
   ~    See the License for the specific language governing permissions and
   ~    limitations under the License.
   -->
-
 <div fxLayout>
   <app-my-sidebar fxFlexAlign="stretch"></app-my-sidebar>
 
+  <div id="row-holder" fxLayout="column" fxLayoutAlign="start start" class="p-3 w-100">
+    <mat-card class="alert alert-info mx-1 w-100" role="alert"> Major Release: </mat-card>
+    <div id="entry-holder" fxLayout="row" fxLayoutGap="10px" class="w-100 mb-4">
+      <mat-card class="mat-card-grow">
+        <div id="header-bar" fxLayout="row" fxLayoutGap="5px" class="mat-card-header-separate">
+          <div id="workflow-number" fxLayout="row">
+            <mat-card-header fxLayout="row">
+              <mat-card-title>Workflows</mat-card-title>
+              <span class="bubble">0</span>
```

Comment:
Will this use an API to get a user's number of workflows instead of being hard-coded as `0`?

Reply:
Yes it will! This is just a placeholder for the visual effect